runtime/lib/ttnn/runtime.cpp

```diff
@@ -61,6 +61,7 @@ Device openDevice(std::vector<int> const &deviceIds,
   assert(deviceIds.size() == 1 && "Only one device is supported for now");
   assert(numHWCQs.empty() && "HWCQs are not supported for now");
   auto &device = ::ttnn::open_device(deviceIds.front(), kL1SmallSize);
+  device.enable_async(true);
```

Comment:
Can you open an issue to do this in emitc path as well, please?

Reply:
Created issue here: #733